src/containers/Navbar/largeScreenNavbarData.js

```diff
@@ -0,0 +1,32 @@
+export const largeScreenNavbarData = [
+  {
+    id: "a",
+    target: "/",
+    name: "Home",
+  },
+  {
```

Comment:
can we move these data files into a seprate folder and call it data ? instead of it being in the container